app.js

```diff
@@ -154,7 +154,8 @@ function handleTheEvent(e){
     }
   }
   currentRound++;
-  if(currentRound === 25){
+  // change back to 25 after fixing rendering
+  if(currentRound === 10){
```

Comment:
There's nothing wrong with this, but I find it easier to remember to change it (and also to find it again) if you store the number in a variable at the top of your page.